plugin.js

```diff
@@ -107,16 +107,16 @@ class WasmPackPlugin {
         let first = true
 
         compiler.hooks.thisCompilation.tap('WasmPackPlugin', (compilation) => {
+            // This is needed in order to gracefully handle errors in Webpack,
+            // since Webpack has its own custom error system.
+            if (this.error != null) {
+                compilation.errors.push(this.error)
+            }
+
             // Super hacky, needed to workaround a bug in Webpack which causes
             // thisCompilation to be triggered twice on the first compilation.
             if (first) {
                 first = false
```

Comment:
if we don't use `first` anymore can you remove it?

Reply:
Sure!